pages/Synthetic-Emerald.md

```diff
@@ -0,0 +1,4 @@
+The Synthetic Emerald is a [Resource](https://github.com/Slimefun/Slimefun4/wiki/Resources) that can be used in place of a Vanilla Minecraft emerald for any purposes.
```

Comment:
```suggestion
The Synthetic Emerald is a [Resource](https://github.com/Slimefun/Slimefun4/wiki/Resources) that can be used instead of a Vanilla Minecraft emerald for all its purposes.
```